.github/workflows/black.yml

```diff
@@ -0,0 +1,10 @@
+name: Black
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
```

Comment:
This version needs to match too. Docs seem to imply this will work:

```
- uses: psf/black@stable
  with:
    version: "22.6.0"
```

Reply:
Arg, sorry --- I totally forgot about this, let me make that fix.